### PR TITLE
flex added to activity page

### PR DESCRIPTION
**Pull request by CFDevelopment**
- flex layout added to activity page

### CODE REVIEW

File: src/screens/main/tokens/history/CoinActivity.js [hunk 1]
```diff
@@ -82,40 +82,50 @@ class CoinActivity extends Component {
...
-          />
-          <CoinSendTabNavigator navigation={this.props.navigation} />
-          <FlatList
-            data={this.state.data}
-            keyExtractor={(x, i) => i.toString()}
-            style={{ flex: 1, width: '100%', backgroundColor: '#fafbfe' }}
-            renderItem={({ item }) => <CardSection>
-                <View style={itemStyle}>
-                  <View>
+
```
Comment (priom) [inline src/screens/main/tokens/history/CoinActivity.js]: remove whitespaces

File: src/screens/main/tokens/receive/CoinReceive.js [hunk 2]
```diff
@@ -30,13 +32,20 @@ class CoinReceive extends Component {
     return (
       <SafeAreaView style={styles.safeAreaView}>
         <View style={mainContainer}>
+
```
Comment (priom) [inline src/screens/main/tokens/receive/CoinReceive.js]: here as well

File: src/screens/main/tokens/receive/CoinReceive.js [hunk 3]
```diff
@@ -71,13 +80,19 @@ const styles = StyleSheet.create({
   },
   mainContainer: {
     flex: 1,
-    alignItems: 'center',
+    // alignItems: 'center',
```
Comment (priom) [inline src/screens/main/tokens/receive/CoinReceive.js]: remove redundant comments

File: src/screens/main/tokens/receive/CoinReceive.js [hunk 4]
```diff
@@ -71,13 +80,19 @@ const styles = StyleSheet.create({
...
   },
+  navContainer: {
+    flex: 0.75,
+  },
+  navHeaderContainer: {
+    flex: 0.75,
+  },
   contentContainer: {
-    marginTop: '10%',
+    // marginTop: '10%',
```
Comment (priom) [inline src/screens/main/tokens/receive/CoinReceive.js]: same here

File: src/screens/main/tokens/history/CoinActivity.js (hunk 1 above)
Comment (CFDevelopment) [reply src/screens/main/tokens/history/CoinActivity.js]: done

**Pull request merged**